packages/app-desktop/integration-tests/markdownEditor.spec.ts

```diff
@@ -63,5 +63,61 @@ test.describe('markdownEditor', () => {
 		await mainWindow.keyboard.press('Home');
 		await expect(firstItemLocator).toBeFocused();
 	});
+
+	test('should sync local search between the viewer and editor', async ({ mainWindow }) => {
+		const mainScreen = new MainScreen(mainWindow);
+		await mainScreen.waitFor();
+		const noteEditor = mainScreen.noteEditor;
+
+		await mainScreen.createNewNote('Note');
+
+		await noteEditor.focusCodeMirrorEditor();
+
+		await mainWindow.keyboard.type('# Testing');
+		await mainWindow.keyboard.press('Enter');
+		await mainWindow.keyboard.press('Enter');
+		await mainWindow.keyboard.type('This is a test of search. `Test inline code`');
+
+		const viewer = noteEditor.getNoteViewerIframe();
+		await expect(viewer.locator('h1')).toHaveText('Testing');
+
+		const matches = viewer.locator('mark');
+		await expect(matches).toHaveCount(0);
+
+		await mainWindow.keyboard.press('Control+f');
```

Comment:
This needs to be `Command` (`Cmd`?) on MacOS. Currently, this is causing a test to fail on the MacOS CI.